src/shared/platforms/ios/iosgleanbridge.swift

```diff
@@ -27,18 +27,20 @@ public class IOSGleanBridgeImpl : NSObject {
           defaults.set(isTelemetryEnabled, forKey: Constants.UserDefaultKeys.telemetryEnabled)
       }
       
-      do {
-            Logger.global?.log(message: "Sending telemetry state change message.");
+      if (TunnelManager.session != nil) {
+        do {
+            Logger.global?.log(message: "Attempting to send telemetry state change message.");
           try TunnelManager.session!.sendProviderMessage(
```

Comment:
```suggestion
      if let session = TunnelManager.session {
        do {
            Logger.global?.log(message: "Attempting to send telemetry state change message.");
          try session.sendProviderMessage(
```

`if let` checks if it's nil, and if not it enters the if block with `session` as a non-optional.